io/module_write_restart_netcdf.F90

```diff
@@ -565,7 +565,12 @@ subroutine write_out_ungridded_dim_atts_from_field(field, dimLabel, dimid, rc)
         ncerr = nf90_def_dim(ncid, trim(dimLabel), valueCount, dimid=dimid); NC_ERR_STOP(ncerr); NC_ERR_STOP(ncerr)
       endif
       if( typekind == ESMF_TYPEKIND_R4 ) then
-        ncerr = nf90_def_var(ncid, trim(dimLabel), NF90_FLOAT, dimids=(/dimid/), varid=varid); NC_ERR_STOP(ncerr)
+        !!! FIXME Use NF90_DOUBLE as axis type, even though axis data are float
+        !!!       This is needed to make phy/sfc restart files identical to FMS
+        !!!       restart files which always defines all axis as double
+
+        ! ncerr = nf90_def_var(ncid, trim(dimLabel), NF90_FLOAT, dimids=(/dimid/), varid=varid); NC_ERR_STOP(ncerr)
+        ncerr = nf90_def_var(ncid, trim(dimLabel), NF90_DOUBLE, dimids=(/dimid/), varid=varid); NC_ERR_STOP(ncerr)
```

Comment:
Is this a temporary solution? Will we allow the the axis type to be float real after the transition from FMS io? I feel that might be consistent with the axis data type defined in the model.

Reply:
@junwang-noaa has your question been satisfied here?

Reply:
@DusanJovic-NOAA can you answer this question?